scripts/mrbind/install_mrbind_ubuntu.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Downloads the source code for MRBind and builds it at `~/mrbind/build`.
+
+set -euxo pipefail
+
+sudo apt install -y git
```

Comment:
Warn user about git dependency instead of silent installing. Something like:
```bash
if ! command -v git 2>&1 >/dev/null ; then
  >&2 echo "git is required"
  exit 1
fi
```

Reply:
I think I'll remove this altogether.